src/v2/Components/UserSettings/UserInformation/index.tsx

```diff
@@ -0,0 +1,232 @@
+              value={values.phone}
+              onChange={handleChange}
+              onBlur={handleBlur}
+            />
+            {me.paddleNumber && (
+              <>
+                <Text>Bidder number</Text>
+                <QuickInput
+                  block
+                  name="paddleNumber"
+                  value={me.paddleNumber}
+                  readOnly
+                />
+              </>
+            )}
+            <input name="internalID" value={me.internalID} hidden readOnly />
+            {touched.email && values.email !== me.email && (
+              <>
+                <Text>Password</Text>
+                <PasswordInput
+                  autoFocus
+                  block
+                  // @ts-expect-error STRICT_NULL_CHECK
+                  error={
+                    !values.password && "Password is required to change email."
+                  }
+                  placeholder="Enter your password"
+                  name="password"
+                  value={values.password}
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  required
+                />
+              </>
+            )}
+            {status && !status.success && (
+              <Banner variant="error">{status.error}</Banner>
+            )}
+            <Button
+              type="submit"
+              size="large"
+              loading={isSubmitting}
+              width="100%"
+              variant="secondaryOutline"
+            >
+              Save changes
+            </Button>
+          </Form>
+        )}
+      </Formik>
+    </Box>
+  )
+}
+
+export const UserInformationRefetchContainer = createRefetchContainer(
+  UserInformation,
+  {
+    me: graphql`
+      fragment UserInformation_me on Me {
+        email
+        name
+        paddleNumber
+        phone
+        internalID
+      }
+    `,
+  },
+  graphql`
+    query UserInformationRefetchQuery {
+      me {
+        ...UserInformation_me
+      }
+    }
+  `
+)
+
+export const UserInformationQueryRenderer = () => {
+  const { user, relayEnvironment } = useSystemContext()
+
+  if (!user) {
+    return null
+  }
+
+  return (
+    <QueryRenderer<UserInformationQuery>
+      // @ts-expect-error STRICT_NULL_CHECK
+      environment={relayEnvironment}
+      variables={{}}
+      query={graphql`
+        query UserInformationQuery @raw_response_type {
+          me {
+            ...UserInformation_me
+          }
+        }
+      `}
+      render={renderWithLoadProgress(UserInformationRefetchContainer)}
+    />
+  )
+}
+
+// @ts-expect-error STRICT_NULL_CHECK
+type GravityFieldErrors = UpdateUserInformationMutationResponse["updateMyUserProfile"]["userOrError"]["mutationError"]
+
+const formatGravityErrors = ({ fieldErrors }: GravityFieldErrors) => {
+  const formatted = {}
+  fieldErrors.map(err => {
+    formatted[err["name"]] = err.message.split(", ")
+  })
+  return formatted
+}
```

Comment:
Old code.

Reply:
This will also fix Integrity for the time being. We'll have to update all the specs once the new page is launched, of course.